src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -191,8 +191,7 @@ public void SarifLogger_ScrapesFilesFromResult()
                         {
                             new Location
                             {
-                                AnalysisTarget = new PhysicalLocation {  Uri = new Uri(@"file:///file0.cpp")},
```

Comment:
Shouldn't `AnalysisTarget` have moved up to the `Result` object?

Reply:
Yep